manifests/params.pp

```diff
@@ -58,4 +58,5 @@
   if $init_style == undef {
     fail('Unsupported OS')
   }
+  notify {"DEBIAN STYLE ${init_style}":}
```

Comment:
It looks like you are working on the same branch while working on the init script thing.